pkgs/development/ocaml-modules/mparser/pcre.nix

```
@@ -0,0 +1,18 @@
+{ fetchFromGitHub, buildDunePackage, ocaml_pcre, mparser }:
+
+buildDunePackage rec {
+  pname = "mparser-pcre";
+  version = "1.3";
+  useDune2 = true;
+
+  src = fetchFromGitHub {
+    owner = "murmour";
+    repo = "mparser";
+    rev = version;
+    sha256 = "16j19v16r42gcsii6a337zrs5cxnf12ig0vaysxyr7sq5lplqhkx";
+  };
```

You can just reuse the source from `mparser`, makes updating easier as well:

```suggestion
  inherit (mparser) src;
```